colab/edge_detection.ipynb

```diff
@@ -3,7 +3,7 @@
   "nbformat_minor": 0,
   "metadata": {
     "colab": {
-      "name": "edge_detection.ipynb",
+      "name": "Resolved_devicearr_ bug_of_edge_detection.ipynb",
```

Comment:
Let's keep the original name here.

The other changes like `"execution_count"` and `"outputId"` are fine to keep.